pulley/src/interp.rs

```diff
@@ -2695,6 +2695,42 @@ impl OpVisitor for Interpreter<'_> {
         self.state[dst].set_u128(val);
         ControlFlow::Continue(())
     }
+
+    fn vsplatx8(&mut self, dst: VReg, src: XReg) -> ControlFlow<Done> {
+        let val = self.state[src].get_u32() as u8;
```

Comment:
We may need `get_u8` and `get_u16` eventually?

Reply:
...just mulling over whether discarding the upper bits here is going to be problematic &mdash; probably not?

Reply:
I've been wondering the same yeah, I'll probably add these in a follow-up